package.json

```diff
@@ -4,7 +4,7 @@
    "private": true,
    "main": "lib/index.js",
    "scripts": {
-      "build": "ncc build src/run.ts -o lib",
+      "build": "npm i ncc && npx ncc build src/run.ts -o lib",
```

Comment:
should installing ncc be a prebuild using npm pre scripts?

Reply:
that's a good point

ideally it would just get installed via dev dependencies and then we can invoke via `npx` but for some reason that fails on gh action runners

i can test a prebuild script on a fork to confirm